packages/client/mutations/toasts/popNotificationToast.ts

```diff
@@ -50,6 +53,23 @@ export const popNotificationToastOnNext: OnNextHandler<
     return
   }
 
+  if (notificationSnack.autoDismiss === 0) {
+    const callback = notificationSnack.onDismiss
+    notificationSnack.onDismiss = () => {
+      const {id: notificationId} = addedNotification
+      SetNotificationStatusMutation(
+        atmosphere,
+        {
+          notificationId,
+          status: 'CLICKED'
```

Comment:
-1 Why was this `onDismiss` event added? It's also sending the same `'CLICKED'` event as the action.